### PR TITLE
.github/workflows: Add CLI installation checks.

### PR DESCRIPTION
**Pull request by kfirtoledo**
Add installation check using the CLI :
This test checks the CLI commands for creating certificates, deploying peers, and deleting them.

### CODE REVIEW

File: tests/cli/basic_test.sh [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+cd $TEST_DIR
+clean_up
+
+trap clean_up_with_logs INT TERM EXIT
+
+cd $TEST_DIR
+test_k8s
+
+echo OK
```
Comment (elevran) [inline tests/cli/basic_test.sh:80]: nit: missing EoL

File: tests/cli/basic_test.sh [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        echo "Error: Timeout occurred while waiting for cl-controlplane deployment"
+        exit 1
+    fi
+
+    if ! timeout 30 sh -c 'until kubectl rollout status deployment cl-dataplane -n clusterlink-system; do sleep 0.1; done > /dev/null 2>&1'; then
+        echo "Error: Timeout occurred while waiting for cl-dataplane deployment"
+        exit 1
+    fi
+
+  # Selete clusterlink objects
```
Comment (elevran) [inline tests/cli/basic_test.sh:67]:
```suggestion
  # Delete clusterlink objects
```

File: .github/workflows/pr-cli.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+name: PR check - CLI Installation test
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  cli-test:
+    runs-on: ${{ matrix.os }}
```
Comment (elevran) [inline .github/workflows/pr-cli.yml:10]: the PR says "for different platforms" but it's only running on ubuntu-latest (IIUC the use of `matrix.os` from below)

**Review by elevran — APPROVED** [review]

File: .github/workflows/pr-cli.yml (hunk 3 above)
Comment (kfirtoledo) [reply .github/workflows/pr-cli.yml:10]: Yes, you are right, I will fix the description.

**Pull request merged**